src/main/resources/org/sonatype/nexus/itscriptclient/blobs/create_blob.groovy

```diff
@@ -0,0 +1,7 @@
+import org.sonatype.nexus.blobstore.api.BlobStore
+
+blobName = "%s"
+existingBlobStore = blobStore.getBlobStoreManager().get(blobName)
+if (existingBlobStore == null) {
+  blobStore.createFileBlobStore(blobName, "/tmp")
+}
```

Comment:
Trivial: add end line for all these files

Reply:
https://github.com/sonatype-nexus-community/nexus-it-script-client/pull/2/commits/4d8d2fec74a1fbbd79696f4eb8cbd3cf1ab53698